templates/windows/nova.conf

```diff
@@ -5,6 +5,8 @@ logging_exception_prefix = '%(asctime)s.%(msecs)03d %(process)d %(thread)d %(thr
 auth_strategy = keystone
 debug = true
 verbose = true
+vif_plugging_is_fatal=false
+vif_plugging_timeout=60

```

Comment:
set timeout lower, 5-10 seconds (waiting 60 seconds might cause some tests to fail)